job-dsls/jobs/kie_release_jobs.groovy

```diff
@@ -1148,6 +1213,7 @@ nestedView("kieReleases-${kieMainBranch}"){
                 name("removeReleaseBranches-kieReleases-${kieMainBranch}")
                 name("updateToNextDevelopmentVersion-kieReleases-${kieMainBranch}")
                 name("copyBinariesToFilemgmt-kieReleases-${kieMainBranch}")
+                name("jBPM-installer-kieReleases-${kieMainBranch}")
```

Comment:
Just thinking out loud: do we need a new job for this? Does it take too long? Or what was the reasoning behind creating a new job instead of updating the `copyBinariesToFilemgmt` one?

Reply:
@psiroky I will remove the job and add a call to kie-jbpmInstaller.sh in copyBinariesToFilemgmt